PayPalWebPayments/src/test/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClientUnitTest.kt

```diff
@@ -36,26 +42,37 @@ import org.robolectric.RobolectricTestRunner
 @RunWith(RobolectricTestRunner::class)
 class PayPalWebCheckoutClientUnitTest {
 
+    @MockK
     private val activity: FragmentActivity = mockk(relaxed = true)
+
+    @MockK
     private val analytics = mockk<PayPalWebAnalytics>(relaxed = true)
-    private lateinit var patchCCOWithAppSwitchEligibility: PatchCCOWithAppSwitchEligibility
-    private lateinit var deviceInspector: DeviceInspector
+
+    @MockK
+    private val patchCCOWithAppSwitchEligibility: PatchCCOWithAppSwitchEligibility =
+        mockk(relaxed = true)
+
+    @MockK
+    private val deviceInspector: DeviceInspector = mockk(relaxed = true)
     private val coreConfig = CoreConfig("fake-client-id", Environment.SANDBOX)
     private val urlScheme = "com.example.app"
     private val fakeAppSwitchUrl = "https://paypal.com/vault-app-switch"
-    private val updateClientConfigAPI = mockk<UpdateClientConfigAPI>(relaxed = true)
+
+    @MockK
+    private val updateClientConfigAPI: UpdateClientConfigAPI = mockk(relaxed = true)
     private val appLinkUrl = "https://example.com/"
 
     private val intent = Intent()
 
-    private lateinit var payPalWebLauncher: PayPalWebLauncher
+    @MockK
+    private val payPalWebLauncher: PayPalWebLauncher = mockk(relaxed = true)
     private lateinit var sut: PayPalWebCheckoutClient
 
+    private val testDispatcher = StandardTestDispatcher()
+
     @Before
     fun beforeEach() {
-        payPalWebLauncher = mockk(relaxed = true)
-        patchCCOWithAppSwitchEligibility = mockk(relaxed = true)
-        deviceInspector = mockk(relaxed = true)
+        Dispatchers.setMain(testDispatcher)
```

Comment:
```suggestion
        MockKAnnotations.init(this)
        Dispatchers.setMain(testDispatcher)
```
We should reset the mocks between every test run.